### ISSUE TITLE
Animated text animations not working when interpolation is on

### ISSUE THREAD

**Issue by MrNooberDude** [PenguinMod/PenguinMod-Home#24]
yeah, not much else really

**Comment by MrNooberDude** [comment]
scratch that, the entire extension just doesn't work :/

**Comment by JoshAtticus** [comment]
Not a PenguinMod problem

**Comment by JeremyGamer13** [comment]
this shouldnt be closed if the issue is still present, regardless of if the issue is also in TW

**Comment by JoshAtticus** [comment]
> this shouldnt be closed if the issue is still present, regardless of if the issue is also in TW

Yes but it does not affect specifically us and should be moved to the Turbowarp-VM repo or the PenguinMod-VM repo
